api/src/main/java/org/openmrs/module/m2sysbiometrics/util/ContextUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.openmrs.module.m2sysbiometrics.util;
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.m2sysbiometrics.exception.M2SysBiometricsException;
+import java.util.List;
+
+public final class ContextUtils {
```

Comment:
what's the point of this, why can't we just autowire the service ?

Reply:
We can, I'll remove it. It should not be used in this class, and I added it to this PR by mistake.